app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :title, presence: true, length: { maximum: 50 }
+  validates :body, presence: true, length: { minimum: 200 }
```

Comment:
Probably instead of specifying a minimum length here you could specify the maximum (as in the title). Users won't find it pleasing for them to have to write at least 200 characters.